cmd/jimmsrv/main.go

```diff
@@ -195,6 +218,14 @@ func start(ctx context.Context, s *service.Service) error {
 		LogSQL:                    logSQL,
 		LogLevel:                  logLevel,
 		IsLeader:                  os.Getenv("JIMM_IS_LEADER") != "",
+		ControllerConfig: config.ControllerConfig{
+			ControllerUUID:              os.Getenv("JIMM_UUID"),
+			PublicDNSName:               os.Getenv("JIMM_DNS_NAME"),
```

Comment:
In the future we should probably do validation on these, but not something for this PR.